staging/src/k8s.io/apiextensions-apiserver/pkg/apiserver/schema/cel/compilation.go

```diff
@@ -125,6 +126,9 @@ func Compile(s *schema.Structural, declType *apiservercel.DeclType, perCallLimit
 	if len(s.XValidations) == 0 {
 		return nil, nil
 	}
+	if declType == nil {
+		return nil, errors.New("Failed to convert to declType for CEL validation rules")
```

Comment:
does returning an error here prevent successful creation of the handler for the CRD in the reproducer?

Reply:
No. See the integration test to verify.